### PR TITLE
Add a tagged flag to EdgeInfo and Sign to support future "tagged" or …

### PR DESCRIPTION
**Pull request by dnesbitt61**

### CODE REVIEW

File: valhalla/baldr/sign.h [hunk 1]
```diff
@@ -26,7 +26,7 @@ class Sign {
    * @param  text_offset  Offset to text in the names/text table.
    */
   Sign(const uint32_t idx, const Sign::Type& type, const bool rn, const uint32_t text_offset)
-      : edgeindex_(idx), type_(static_cast<uint32_t>(type)), is_route_num_(rn), spare_(0),
+      : edgeindex_(idx), type_(static_cast<uint32_t>(type)), is_route_num_(rn), tagged_(0),
```
Comment (dgearhart) [inline valhalla/baldr/sign.h]: did you still want to zero out spare?

Comment (dgearhart) [reply valhalla/baldr/sign.h]: oh - ignore, i was looking at wrong spare count

Comment (dnesbitt61) [reply valhalla/baldr/sign.h]: no more spare!

File: valhalla/baldr/edgeinfo.h [hunk 2]
```diff
@@ -39,7 +39,12 @@ struct NameInfo {
...
                                    // vs just a name.
-  uint32_t spare_ : 3;
+  uint32_t tagged_ : 1;            // Future use - this indicates the text string is
+                                   // specially tagged (for example uses the first char as
+                                   // the tag type). To make this forward and backward
+                                   // compatible, tagged text will not be read in GetNames
+                                   // and GetNamesAndTags until code is ready to actually use it.
+  uint32_t spare_ : 2;
 
   bool operator==(const NameInfo& other) const {
```
Comment (dgearhart) [inline valhalla/baldr/edgeinfo.h]: what about the equals and less than operators? They were not updated with is_route_num yet either

**Comment by dgearhart** [comment]
do we ignore logic in `EdgeInfo::GetTypes()` too?

**Pull request merged**